### PR TITLE
Empty as null

### PR DESCRIPTION
**Pull request by muga**

### CODE REVIEW

File: src/main/java/com/treasure_data/td_import/Constants.java [hunk 1]
```diff
@@ -95,6 +95,7 @@
             "    --column-types TYPE,TYPE,...     column types [string, int, long, double]\n" +
             "    --column-type NAME:TYPE          column type [string, int, long, double]. A pair of column name and type can be specified like 'age:int'\n" +
             "    -S, --all-string                 disable automatic type conversion\n" +
+            "    --empty-as-null                  the empty string values are interpreted as null values if columns arenumerical types." +
```
Comment (nahi) [inline src/main/java/com/treasure_data/td_import/Constants.java]: "empty-as-null" sounds it works for String type as well (but not true I assume.) How about --empty-numeric-as-null?

Comment (nahi) [inline src/main/java/com/treasure_data/td_import/Constants.java]: typo: arenumerical -> are numerical

Comment (muga) [inline src/main/java/com/treasure_data/td_import/Constants.java]: > "empty-as-null" sounds it works for String type as well (but not true I assume.) How about --empty-numeric-as-null?

Right, 'empty-as-null' is not correct. To be more precise, the option is `empty-string-as-null-if-numeric`. 'empty-numeric' is also not correct.

Anyway, the origin of the name is Postgresql's copy option: http://www.postgresql.org/docs/9.4/static/sql-copy.html. I think that 'empty-as-null' is better instead.

Comment (muga) [inline src/main/java/com/treasure_data/td_import/Constants.java]: good catch > typo

Comment (nahi) [inline src/main/java/com/treasure_data/td_import/Constants.java]: I don't see neither the name 'empty-as-null' nor numeric value handling in the page... How does the name come from?

Comment (muga) [inline src/main/java/com/treasure_data/td_import/Constants.java]: sorry. it came from Redshift http://docs.aws.amazon.com/redshift/latest/dg/r_COPY.html

Comment (nahi) [inline src/main/java/com/treasure_data/td_import/Constants.java]:
```
Indicates that Amazon Redshift should load empty CHAR and VARCHAR fields as NULL. Empty fields for other data types, such as INT, are always loaded with NULL.
```
My understanding is 'emptyasnull' is for CHAR and VARCHAR (String) and it does not affect other data types such as INT, FLOAT, DOUBLE. That's exactly what I pointed that the name would not be good.

Comment (muga) [inline src/main/java/com/treasure_data/td_import/Constants.java]: That's right. I will rename 'empty-as-null-if-numeric':-)

File: src/main/java/com/treasure_data/td_import/Constants.java [hunk 2]
```diff
@@ -295,6 +296,12 @@
     String BI_PREPARE_ALL_STRING_HYPHEN = HYPHENHYPHEN + BI_PREPARE_PARTS_COLUMNS;
     String BI_PREPARE_ALL_STRING_DESC = "disable automatic type conversion";
 
+    // empty-as-null-if-numeric; default=false
+    String BI_PREPARE_PARTS_EMPTYASNULL = "empty-as-null-if-numeric";
+    String BI_PREPARE_EMPTYASNULL_HYPHEN = HYPHENHYPHEN + BI_PREPARE_PARTS_EMPTYASNULL;
+    String BI_PREPARE_PARTS_EMPTYASNULL_DEFAULTVALUE = "false";
+    String BI_PREPARE_PARTS_EMPTYASNULL_DESC = "the empty string values are interpreted as null values if columns arenumerical types";
```
Comment (nahi) [inline src/main/java/com/treasure_data/td_import/Constants.java]: Oh I found one more 'arenumerical' typo.

File: CHANGES.txt [hunk 3]
```diff
@@ -2,6 +2,7 @@ Release 0.5.7 - 20XX/XX/XX
   IMPROVEMENTS
     MySQLTableReader supported MEDIUMINT and MEDIUMINT UNSIGNED.
     Upgrades aws-java-sdk 1.7.5
+    Added new '--empty-as-null' option to 'import:prepare' command
```
Comment (nahi) [inline CHANGES.txt]: typo: should be --empty-as-null-if-numeric

Comment (muga) [inline CHANGES.txt]: thanks, i fix it

File: src/main/java/com/treasure_data/td_import/Constants.java (hunk 2 above)
Comment (muga) [inline src/main/java/com/treasure_data/td_import/Constants.java]: :-(

**Pull request merged**